agents/smurf_file_emulator/smurf_file_emulator.py

```diff
@@ -186,6 +194,8 @@ def get_data_frame(self, start, stop):
 
         times = np.arange(start, stop, 1. / self.sample_rate)
         nsamps = len(times)
+        frame_counter = np.arange(self.sample_num, self.sample_num + nsamps, dtype=int)
+        self.sample_num += nsamps
```

Comment:
Is `self.sample_num` going to be used elsewhere later?

Reply:
`get_data_frame` can be called multiple times per streaming session, so `self.sample_num` needs to be persistent / kept up-to-date

Reply:
Ah, gotcha, thanks!